recipe/run_test.py

```diff
@@ -382,21 +270,11 @@ def check_libraries(libraries_files):
     check_outputs(nnet2_bins)
     check_outputs(nnet3_bins)
     check_outputs(rnnlm_bins)
+    check_outputs(online2_bins)
 
     if test_cuda:
         check_outputs(cuda_decoder_bins)
         check_outputs(cuda_feat_bins)
 
     if sys.platform != 'win32':
         check_outputs(online_bins)
-        check_outputs(online2_bins)
```

Comment:
I find it kinda curious that we now have `online2_bins` on windows, but not `online_bins`. Not that I'm complaining (I don't know kaldi nearly well enough), but out of curiosity, what would still be missing to enable those also on windows?

Reply:
I think I might be able to enable it, I think it was disabled still because most of the online depends on tcp.h, but it looks like there are two binaries that should be buildable in windows, so I'll get those added.